hackster-service/src/test/java/com/lohika/jclub/HacksterServiceApplicationTests.java

```diff
@@ -2,14 +2,50 @@
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
+@AutoConfigureMockMvc
 public class HacksterServiceApplicationTests {
+  @Value("${maxAllowedApartmentsPerRealtor}")
+  private int maxAllowedApartmentsPerRealtor;
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @Test
+  public void checkIfNewPhoneNumberIsNotHakster() throws Exception {
```

Comment:
Not agree with Snake_Case, if you don't like this name we can discuss it, but change Java convention to something else doesn't make sense